packages/icons/web/v2/jcb_32.svg

```diff
@@ -0,0 +1,48 @@
+<svg width="32" height="24" viewBox="0 0 32 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I asked [a question about this one](https://www.figma.com/file/vnFVuPNlqF45rkw1u9toBC?node-id=4135:19#121332846) on Figma, because it looks slightly weird there

Reply:
There's a new one without the grey border.